tests/testRepository.py

```diff
@@ -147,6 +145,19 @@ def testRaw(self):
         self.assertAlmostEqual(weather.getAirPressure(), self.weath_airPressure)
         self.assertAlmostEqual(weather.getHumidity(), self.weath_humidity)
 
+    def testPupil(self):
+        """Test retrieval of pupil (without checking value)"""
```

Comment:
Don't you care that it's at least an `HscPupilFactory`?  That seems relatively easy to test.